lib/rspecq/queue.rb

```diff
@@ -125,7 +126,7 @@ def acknowledge_job(job)
     # Returns nil if the job hit the requeue limit and therefore was not
     # requeued and should be considered a failure.
     def requeue_job(job, max_requeues)
-      return false if max_requeues.zero?
+      return false if max_requeues.zero? || build_failed_fast?
```

Comment:
Why is this needed? Workers will anyway exit on their own if `build_failed_fast? == true`.